File: result/translate/chatgpt/java2python/python/FIND_SUBARRAY_WITH_GIVEN_SUM.py
def f_gold(arr, n, sum):
    for i in range(n):
        curr_sum = arr[i]
        for j in range(i + 1, n + 1):
            if curr_sum == sum:
                p = j - 1
                print("Sum found between indexes " + str(i) + " and " + str(p))
                return 1
            if curr_sum > sum or j == n:
                break
            curr_sum += arr[j]
    print("No subarray found")
    return 0

File: dataset/java2python/java/FIND_SUBARRAY_WITH_GIVEN_SUM.java
int f_gold ( int arr [ ] , int n , int sum ) {
  int curr_sum , i , j ;
  for ( i = 0 ;
  i < n ;
  i ++ ) {
    curr_sum = arr [ i ] ;
    for ( j = i + 1 ;
    j <= n ;
    j ++ ) {
      if ( curr_sum == sum ) {
        int p = j - 1 ;
        System . out . println ( "Sum found between indexes " + i + " and " + p ) ;
        return 1 ;
      }
      if ( curr_sum > sum || j == n ) break ;
      curr_sum = curr_sum + arr [ j ] ;
    }
  }
  System . out . println ( "No subarray found" ) ;
  return 0 ;
}